src/components/Calendar.js

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+import {useState} from 'react';
+import Calendar from 'react-calendar';
+
+import {CalenderContainer} from '../components/styled.components/Calender.styled';
+
+export default function App() {
+	const [dateState, setDateState] = useState(new Date());
+	const changeDate = e => {
+		setDateState(e);
+	};
+	return (
+		<CalenderContainer>
+			<Calendar value={dateState} onChange={changeDate} />
+			Current selected date is <b>{moment(dateState).format('MMMM Do YYYY')}</b>
```

Comment:
```suggestion
			Current selected date is <b>{moment(dateState).format('DD.MM.YYYY')}</b>
```

would be the better format for our region ;)